cdn/agent-aggregator/util/features.js

```diff
@@ -14,7 +17,7 @@ export const modules = {
   metrics: 'metrics'
 }
 
-const lite = [modules.pageViewEvent, modules.pageViewTiming, modules.metrics]
+const lite = [modules.pageViewEvent, modules.pageViewTiming] // lite agent cannot currently use metrics endpoint, because legacy lite customers are not provisioned access to that endpoint
 const pro = [...lite, modules.jsErrors, modules.ajax, modules.pageAction, modules.sessionTrace]
```

Comment:
You'll need to add metrics to Pro, to retain it for Pro & SPA, or they won't have them either.